### PR TITLE
enhancement/nopd_cprr

### PR DESCRIPTION
**Pull request by ayyubibrahimi**

### CODE REVIEW

**Comment by ayyubibrahimi** [comment]
@pckhoi I'm receiving the error below when running make. My device is indeed low on space. Is this the only issue? 

```
badger 2022/06/02 20:40:02 ERROR: Received err: Open existing file: "/mnt/c/Users/ayyubi/lleads/processing/.wrgl/kv/000678.vlog" error: Create a new file
github.com/dgraph-io/ristretto/z.init
        /Users/runner/go/pkg/mod/github.com/dgraph-io/ristretto@v0.1.0/z/file.go:36
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6498
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.main
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:238
runtime.goexit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/asm_amd64.s:1581. Cleaning up...
During db.vlog.open error: Open existing file: "/mnt/c/Users/ayyubi/lleads/processing/.wrgl/kv/000678.vlog" error: Create a new file
github.com/dgraph-io/ristretto/z.init
        /Users/runner/go/pkg/mod/github.com/dgraph-io/ristretto@v0.1.0/z/file.go:36
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6498
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.doInit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:6475
runtime.main
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:238
runtime.goexit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/asm_amd64.s:1581
badger 2022/06/02 20:40:07 ERROR: Received err: Error while creating log file in valueLog.open error: while opening file: /mnt/c/Users/ayyubi/lleads/processing/.wrgl/kv/000679.vlog error: truncate /mnt/c/Users/ayyubi/lleads/processing/.wrgl/kv/000679.vlog: no space left on device
error while truncation
github.com/dgraph-io/ristretto/z.OpenMmapFileUsing
        /Users/runner/go/pkg/mod/github.com/dgraph-io/ristretto@v0.1.0/z/file.go:50
github.com/dgraph-io/ristretto/z.OpenMmapFile
        /Users/runner/go/pkg/mod/github.com/dgraph-io/ristretto@v0.1.0/z/file.go:86
github.com/dgraph-io/badger/v3.(*logFile).open
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/memtable.go:556
github.com/dgraph-io/badger/v3.(*valueLog).createVlogFile
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/value.go:519
github.com/dgraph-io/badger/v3.(*valueLog).open
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/value.go:619
github.com/dgraph-io/badger/v3.Open
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/db.go:368
github.com/wrgl/wrgl/pkg/local.(*RepoDir).openBadger
        /Users/runner/work/wrgl/wrgl/pkg/local/repo_dir.go:76
github.com/wrgl/wrgl/pkg/local.(*RepoDir).OpenObjectsStore
        /Users/runner/work/wrgl/wrgl/pkg/local/repo_dir.go:80
github.com/wrgl/wrgl/cmd/wrgl.exportCommit
        /Users/runner/work/wrgl/wrgl/cmd/wrgl/export_cmd.go:43
github.com/wrgl/wrgl/cmd/wrgl.newExportCmd.func1
        /Users/runner/work/wrgl/wrgl/cmd/wrgl/export_cmd.go:30
github.com/spf13/cobra.(*Command).execute
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852
github.com/spf13/cobra.(*Command).ExecuteC
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960
github.com/spf13/cobra.(*Command).Execute
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main
        /Users/runner/work/wrgl/wrgl/wrgl/main.go:16
runtime.main
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:255
runtime.goexit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/asm_amd64.s:1581. Cleaning up...
During db.vlog.open error: Error while creating log file in valueLog.open error: while opening file: /mnt/c/Users/ayyubi/lleads/processing/.wrgl/kv/000679.vlog error: truncate /mnt/c/Users/ayyubi/lleads/processing/.wrgl/kv/000679.vlog: no space left on device
error while truncation
github.com/dgraph-io/ristretto/z.OpenMmapFileUsing
        /Users/runner/go/pkg/mod/github.com/dgraph-io/ristretto@v0.1.0/z/file.go:50
github.com/dgraph-io/ristretto/z.OpenMmapFile
        /Users/runner/go/pkg/mod/github.com/dgraph-io/ristretto@v0.1.0/z/file.go:86
github.com/dgraph-io/badger/v3.(*logFile).open
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/memtable.go:556
github.com/dgraph-io/badger/v3.(*valueLog).createVlogFile
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/value.go:519
github.com/dgraph-io/badger/v3.(*valueLog).open
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/value.go:619
github.com/dgraph-io/badger/v3.Open
        /Users/runner/go/pkg/mod/github.com/dgraph-io/badger/v3@v3.2103.1/db.go:368
github.com/wrgl/wrgl/pkg/local.(*RepoDir).openBadger
        /Users/runner/work/wrgl/wrgl/pkg/local/repo_dir.go:76
github.com/wrgl/wrgl/pkg/local.(*RepoDir).OpenObjectsStore
        /Users/runner/work/wrgl/wrgl/pkg/local/repo_dir.go:80
github.com/wrgl/wrgl/cmd/wrgl.exportCommit
        /Users/runner/work/wrgl/wrgl/cmd/wrgl/export_cmd.go:43
github.com/wrgl/wrgl/cmd/wrgl.newExportCmd.func1
        /Users/runner/work/wrgl/wrgl/cmd/wrgl/export_cmd.go:30
github.com/spf13/cobra.(*Command).execute
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:852
github.com/spf13/cobra.(*Command).ExecuteC
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:960
github.com/spf13/cobra.(*Command).Execute
        /Users/runner/go/pkg/mod/github.com/spf13/cobra@v1.1.3/command.go:897
main.main
        /Users/runner/work/wrgl/wrgl/wrgl/main.go:16
runtime.main
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/proc.go:255
runtime.goexit
        /Users/runner/hostedtoolcache/go/1.17.9/x64/src/runtime/asm_amd64.s:1581
make: *** [wrgl.mk:4: pull_person] Error 1
```

**Comment by pckhoi** [comment]
Yeah, looks like space is the only issue. Try to remove everything except the config file in the `.wrgl` folder. Run `wrgl init`. Then run make again and let me know how it goes. Hopefully, it will just pull the person table.

**Comment by ayyubibrahimi** [comment]
No luck after removing everything except the config file in the `.wrgl` folder, running `wrgl init`, then running `make` again. I'll work on my space issue.

**Comment by pckhoi** [comment]
If you cannot easily fix the space situation, let me know. I can probably
cook up some incantations that pull only the person table and nothing else.

**Comment by ayyubibrahimi** [comment]
Go for it. I freed up a bunch of space and am still having no luck.

**Comment by ayyubibrahimi** [comment]
Table pulled sucessfully. Ready for review @pckhoi

File: datavalid.yml [hunk 1]
```diff
@@ -168,6 +168,7 @@ schemas:
       - name: disposition_desc
         description: >
           text description of aftermath of disposition, or reason for disposition.
+      - name: item_number
```
[the remark this answers is not in the document]
Comment (ayyubibrahimi) [reply datavalid.yml:171]: Working on it.

File: fuse/new_orleans_pd.py [hunk 2]
```diff
@@ -184,8 +171,6 @@ def fuse_events(pprr_ipm, pprr_csd, cprr, uof, award, lprr, pclaims20, pclaims21
     pprr_ipm = pd.read_csv(deba.data("match/pprr_new_orleans_ipm_iapro_1946_2018.csv"))
     pprr_csd = pd.read_csv(deba.data("match/pprr_new_orleans_csd_2014.csv"))
     officer_number_dict = create_officer_number_dict(pprr_ipm)
-    cprr = pd.read_csv(deba.data("clean/cprr_new_orleans_pd_1931_2020.csv"))
-    actions = pd.read_csv(deba.data("clean/cprr_actions_new_orleans_pd_1931_2020.csv"))
```
[the remark this answers is not in the document]
Comment (ayyubibrahimi) [reply fuse/new_orleans_pd.py:188]: We received this from a third-party. Prior to cleaning, the table contained three `disposition` columns that were inconsistent, despite the fact that they should be consistent. @rajivsinclair agreed that this cprr dataset should be dropped for this smaller but higher quality dataset. That said, something just came to mind which makes me think that we actually should maybe not drop it just yet.

[the remark this answers is not in the document]
Comment (pckhoi) [reply fuse/new_orleans_pd.py:188]: Okay, let me know when I should review again.

**Pull request closed without merging**